### PR TITLE
Mass delete of unused files

### PR DESCRIPTION
**Pull request by fheysel**
I've deleted all the unused files I could see. @charasyn @jDenyes @allisonchristensen please look through as well as I was not always sure about some of the things you guys implemented.

### CODE REVIEW

File: godotproject/project.godot [hunk 1]
```diff
@@ -17,7 +17,6 @@ _global_script_class_icons={
 
 config/name="Frog Goes To Space - Cooper Prototype"
 run/main_scene="res://Menus/MainMenu/MainMenu.tscn"
-config/icon="res://icon.png"
```
Comment (fheysel) [inline godotproject/project.godot:20]: Oh I deleted the godot icon, I didnt realize it was in use here should I add it back in?

Comment (charasyn) [reply godotproject/project.godot:20]: If it gives errors without it, I'd say add it back in

**Review by jDenyes — APPROVED** [review]

File: godotproject/project.godot (hunk 1 above)
Comment (fheysel) [reply godotproject/project.godot:20]: added back in, no more errors.

**Review by charasyn — APPROVED** [review]
Looks good. Just doublecheck that everything still looks right! 👍

**Pull request merged**